Grievance_Backend/grievance/src/main/java/com/grievance/controller/TicketController.java

```diff
@@ -37,21 +48,63 @@ public class TicketController {
    */
   @PostMapping("/addTicket")
   public ResponseEntity<?> saveTicket(
```

Comment:
why we are not giving which Object will get return in ResponseEntity instead of ?.